app/src/main/res/values/strings.xml

```diff
@@ -1,22 +1,17 @@
 <resources>
     <string name="app_name">Goose Eye</string>
     <string name="button_back_text">Back</string>
-    <string name="pg_text">13+</string>
-    <string name="name_text">Avengers:\nEnd Game</string>
+    <string name="pg_text">+</string>
+    <string name="name_avengers">Avengers:\nEnd Game</string>
     <string name="tag_line_text">Action, Adventure, Fantasy</string>
-    <string name="review_text">125 REVIEWS</string>
+    <string name="review_text">REVIEWS</string>
```

Comment:
I don't think it is a good idea to use caps in strings. I would like to advise this attribute to textview: `android:textAllCaps="true"`
Could you also specify plurals? (one -> review, others -> reviewS)
In other ways your code is pretty great!